hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/domain/RecordItemBuilder.java

```diff
@@ -726,6 +727,13 @@ public Builder<TokenMintTransactionBody.Builder> tokenMint(TokenType tokenType)
         return builder;
     }
 
+    public Builder<TokenUpdateNftsTransactionBody.Builder> tokenNftsUpdate() {
+        var transactionBody = TokenUpdateNftsTransactionBody.newBuilder().setToken(tokenId());
+        transactionBody.addSerialNumbers(1L).addSerialNumbers(2L);
+        transactionBody.setMetadata(BytesValue.of(bytes(16)));
```

Comment:
Prefer a single chained builder like the others and sort.
```suggestion
        var transactionBody = TokenUpdateNftsTransactionBody
            .newBuilder()
            .addSerialNumbers(1L)
            .addSerialNumbers(2L)
            .setMetadata(BytesValue.of(bytes(16)))
            .setToken(tokenId());
```

Reply:
Done